vortex-scalar/src/primitive.rs

```diff
@@ -165,3 +189,11 @@ impl TryFrom<&Scalar> for usize {
         .map(|v| v as Self)
     }
 }
+
+impl TryFrom<Scalar> for usize {
+    type Error = VortexError;
+
+    fn try_from(value: Scalar) -> Result<Self, Self::Error> {
+        usize::try_from(&value)
+    }
+}
```

Comment:
what uses this?

Reply:
This is a drive by. I will move it to separate pr